test/e2e/extensions/high_availability_test.go

```diff
@@ -41,16 +41,22 @@ func TestGameServerHealthyAfterDeletingPodWhileOneExtensionsDown(t *testing.T) {
 
 	list, err := getAgoneseExtensionsPods(ctx)
 	logger.Infof("Length of pod list is %v", len(list.Items))
+	for i := range list.Items {
+		logger.Infof("Name of extensions pod %v: %v", i, list.Items[i].ObjectMeta.Name)
+		logger.Infof("Host IP %v", list.Items[i].Status.HostIP)
+		logger.Infof("Pod IPs %v", list.Items[i].Status.PodIPs)
+	}
 	assert.NoError(t, err, "Could not get list of Extension pods")
```

Comment:
```suggestion
	require.NoError(t, err, "Could not get list of Extension pods")
```

Since if there is an error, the next line will panic, and the list is probably nil

Reply:
Done